app/upgrades/v7/constants.go

```diff
@@ -1,8 +1,10 @@
+//go:build upgrade_v7
+
 package v7
 
 import (
 	store "github.com/cosmos/cosmos-sdk/store/types"
-	icahosttypes "github.com/cosmos/ibc-go/v4/modules/apps/27-interchain-accounts/host/types"
+	icahosttypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types"
 
```

Comment:
v4 is correct ?

Reply:
True, but  IBC v4 causes dependencies conflicts so since these migration scripts are only here for historical reasons, and if needed anyone can use the correct binary version for sequential upgrades.